scripts/build.js

```diff
@@ -168,3 +168,45 @@ function getDisabledTasks(process, defaultDisabled = []) {
 
   return defaultDisabled;
 }
+
+/////////////////////////////////////////////////////////////////////////////////////////////////////
+//// Artifact helper functions
```

Comment:
This should probably all get moved to a file under `/scripts/tasks/size-audit.js`. In that file, you'll say:

```JavaScript
exports.generateSizeData = function() {
  // ... the code that's currently in "executeTask"
}
```

to make sure that the function gets exposed.

And then, in this file (`build.js`), in the `.then( ... )` block above, change it to say:

```JavaScript
.then(() => {
  if (process.env['CI_BUILD']) {
    const { generateSizeData } = require('./tasks/size-audit');
    generateSizeData();
  }
});
```

That way, the file only gets imported if it's going to be used.